wallet-services/src/main/java/org/exoplatform/wallet/storage/TransactionStorage.java

```diff
@@ -171,13 +171,15 @@ private List<TransactionEntity> getWalletTransactions(long networkId,
                                                                                       administration);
     // filter transactions having same Nonce and keep last ones
     Map<Long, TransactionEntity> filterTransactions = new HashMap<>();
+   List <TransactionEntity> filteredTransactions = new ArrayList<>();
```

Comment:
IMO No need to add the list, otherwise we will display transactions that have the same nonce in the same list.